frontend/src/interfaces/uiConfig.ts

```diff
@@ -29,20 +29,16 @@ export interface IProclamationToast {
 }
 
 export interface IFlags {
-    C: boolean;
     P: boolean;
     E: boolean;
     RE: boolean;
     EEA?: boolean;
-    OIDC?: boolean;
```

Comment:
I'm surprised this isn't in use, but this is perhaps a leftover from implementing OIDC as an alternative to Google login.